requirements.txt

```diff
@@ -0,0 +1,4 @@
+flake8
+pytest
```

Comment:
don't put flake8 and pytest in the requirement.txt, install these by hand in your .travis.ym in the install section of that file